decidim-initiatives/db/migrate/20181212155740_add_extra_fields_legal_information_to_initiatives_types.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddExtraFieldsLegalInformationToInitiativesTypes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_initiatives_types, :extra_fields_legal_information, :jsonb
```

Comment:
Maybe add `default: {}` here? I leave this to your opinion 😄

Reply:
I'm not sure. I have seen in other migration files that columns for translated attributes don't have a default value, the {} default is mainly used for attributes containing settings or other kind of metadata, except maybe `decidim_organizations.id_documents_explanation_text`. In some cases a `null: false` constraint is included for translated attributes, but that's not general.

Reply:
Ok, let's leave it as it is then!